pilot/pkg/networking/core/v1alpha3/listener.go

```diff
@@ -159,9 +159,9 @@ var (
 	// We need to forward these ALPNs to upstream so that the upstream can
 	// properly use a HTTP or TCP listener
 	plaintextHTTPALPNs = []string{"http/1.0", "http/1.1", "h2c"}
-	mtlsHTTPALPNs      = []string{"istio-http/1.0", "istio-http/1.1", "istio-h2"}
+	mtlsHTTPALPNs      = []string{"istio-http/1.0", "istio-http/1.1", "istio-h2", "istio-mixerless"}
```

Comment:
This is not the place to inject the alpn. You should add those to `mtlsHTTP10ALPN`, etc.